### ISSUE TITLE
How to bind Polymer's custom event in Angular 2?

### ISSUE THREAD

**Issue by gutenye** [angular/angular#3975]
I want to use Polymer's components in Angular, given
```
<paper-dialog  (^iron-overlay-closed)="onClose()">
...
</paper-dialog>
```

It will not work, for 

>  as angular 2 converts a-b event listner to camel case aB and that’s not what polymer wants…

quote from https://discventionstech.wordpress.com/2015/06/24/using-polymer-1-0-with-angular-2-0/

What's the current state of this and is there other ways to archive it?

**Comment by ericmartinezr** [comment]
Probably related https://github.com/angular/angular/issues/3929

**Comment by pkozlowski-opensource** [comment]
This will be solved by #4007. Closing as a duplicate.
